src/custom/components/AccountDetails/Transaction/index.tsx

```diff
@@ -339,15 +106,24 @@ function ActivitySummary(params: {
   const isOrder = type === ActivityType.ORDER
 
   if (isOrder) {
-    const { inputToken, sellAmount, feeAmount, outputToken, buyAmount, validTo, kind } = activity as Order
+    const { inputToken, sellAmount, feeAmount, outputToken, buyAmount, validTo, kind, fulfillmentTime } =
+      activity as Order
 
     const sellAmt = CurrencyAmount.fromRawAmount(inputToken, sellAmount.toString())
     const feeAmt = CurrencyAmount.fromRawAmount(inputToken, feeAmount.toString())
     const outputAmount = CurrencyAmount.fromRawAmount(outputToken, buyAmount.toString())
+    const limitPrice = formatSmart(getLimitPrice(activity as Order))
+    // const executedPrice = formatSmart(getExecutedPrice(activity as Order))
```

Comment:
I thought Anxo's PR was merged here?
Confused now...

Reply:
Nevermind, I saw it's not merged yet.
Let me know if you need any help.

Reply:
I can use help with checking how we can get the executed price from the API. @anxolin created in the `order` type a field `apiAdditionalInfo` (from `order.apiAdditionalInfo`), which currently isn't getting/storing data from the API yet. Please see https://github.com/gnosis/cowswap/pull/1279/files#diff-8459ecb731c931512504bc667660d0e16d153c2e3af36970f639feeb18b1a2bfR127

Reply:
Addressed in a new PR https://github.com/gnosis/cowswap/pull/1309